pallets/core_mods/src/modules/offchain_signatures/schemas.rs

```diff
@@ -0,0 +1,176 @@
+use crate::{offchain_signatures::SignatureParams, types::CurveType, util::Bytes};
```

Comment:
Change the file name and other occurrences of the word "schema" (in this context) to "scheme".